### ISSUE TITLE
Implement --subprocesses on Windows

### ISSUE THREAD

**Issue by janpio** [rbspy/rbspy#189]
```
λ rbspy record --subprocesses -- ruby c:\Ruby24-x64\bin\bundle exec fastlane -v
Error. Causes:
- --subprocesses option is not available on windows
```

Reason and "workaround" here: https://github.com/rbspy/rbspy/issues/183#issuecomment-424525866

**Comment by akhramov** [comment]
I kind of implemented this in #224, however, tests fail on x86_64 arch. I cannot reproduce this situation locally, so I would appreciate if whoever is interested volunteers me in narrowing the problem down.

You just need to launch tests, as in `cargo test` and report details if tests fail.

Needless to say I'd be happy if someone comes up with an alternative working solution.


PS: I shouldn't have force-pushed the commit with the issue id that often

**Comment by liaden** [comment]
This was fixed in #224
